build/common.sh

```diff
@@ -27,8 +27,10 @@ unset CDPATH
 USER_ID=$(id -u)
 GROUP_ID=$(id -g)
 
+DOCKER_CMD=${DOCKER:-docker}
```

Comment:
This should be something like `CONTAINER_RUNTIME` from my point of view.

Reply:
You mean the name of the environment variable, instead of `DOCKER_CMD`?

Container runtime is something `docker`/`podman` invoke, like `runc`. Here we only try to remove the dependency on the hardcoded `docker` "entrypoint" to the whole container mgmt chain.

Reply:
But I can remove that variable completely and just go with

```diff
 DOCKER_OPTS=${DOCKER_OPTS:-""}
-IFS=" " read -r -a DOCKER <<< "docker ${DOCKER_OPTS}"
+IFS=" " read -r -a DOCKER <<< "${DOCKER:-docker} ${DOCKER_OPTS}"
 DOCKER_HOST=${DOCKER_HOST:-""}
```
to avoid the decision how to name it. :-)

Reply:
Sounds good to me!

Reply:
Updated in 00e2a265bd6a1714b13e875f8dd5fa5fa3059942.